src/filters/turndown.js

```diff
@@ -5,5 +5,41 @@ module.exports = value => {
 		headingStyle: "atx",
 		codeBlockStyle: "fenced"
 	});
+
+	td.keep(["table"]);
+
+	td.addRule("youtube", {
+		filter: ["iframe"],
+		replacement: (innerHTML, node) => {
+			return `{% youtube "${node.getAttribute("src").replace("?feature=oembed", "")}" %}`;
+		}
+	});
+
+	td.addRule("image-and-text", {
+		filter: (node) => {
+			return node.classList.contains("wp-block-media-text");
```

Comment:
Can we take the chance to rename `wp-` classes to get rid of the trace of the historical WordPress? Thanks.

Reply:
This filter will actually be removed as it's only used to find the `wp-` classes in the source content and transform them. I've made the necessary changes in the shortcodes and stylesheets.